Author: Daniel Leidert <dleidert@debian.org>
Bug-Debian: http://bugs.debian.org/681116
Description: Omit any date to the output file to handle
 source files not setting a date/pubdate as quick solution
 to #681116 to get a consistent manpage output in multiarch
 enabled packages independent from the build-time.
Forwarded: no

--- /dev/null
+++ b/docbook-xsl-ns/manpages/docbook-multiarch.xsl
@@ -0,0 +1,18 @@
+<?xml version='1.0'?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                version='1.0'>
+
+<!-- ********************************************************************
+     Customization layer for building Debian Multi-Arch packages omitting
+     the build-tile date!
+     ******************************************************************** -->
+
+<!-- ==================================================================== -->
+
+<xsl:import href="docbook.xsl"/>
+
+<xsl:template name="get.refentry.date">
+  <xsl:value-of select="''"/>
+</xsl:template>
+
+</xsl:stylesheet>
--- /dev/null
+++ b/docbook-xsl/manpages/docbook-multiarch.xsl
@@ -0,0 +1,18 @@
+<?xml version='1.0'?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                version='1.0'>
+
+<!-- ********************************************************************
+     Customization layer for building Debian Multi-Arch packages omitting
+     the build-tile date!
+     ******************************************************************** -->
+
+<!-- ==================================================================== -->
+
+<xsl:import href="docbook.xsl"/>
+
+<xsl:template name="get.refentry.date">
+  <xsl:value-of select="''"/>
+</xsl:template>
+
+</xsl:stylesheet>
